### PR TITLE
Fixes #679 AppVeyor not working

### LINKED ISSUE #679: AppVeyor not working
For example, see:
https://ci.appveyor.com/project/open-systems-pharmacology-ci/ospsuite-r/builds/41802062

Not sure why, but I _think_ this is the source of the problem:

```
C:/projects/ospsuite-r/rakefile.rb:9: warning: already initialized constant OpenSSL::SSL::VERIFY_PEER
rake aborted!
Errno::EINVAL: Invalid argument @ rb_sysopen - https://ci.appveyor.com/api/projects/open-systems-pharmacology-ci/pk-sim/artifacts/setup.zip?branch=develop
```

### PR DESCRIPTION
**Pull request by Yuri05**

### CODE REVIEW

**Comment by PavelBal** [comment]
@IndrajeetPatil Could you check it for esqlabsr?

**Comment by msevestre** [comment]
Waht the heck?

**Comment by msevestre** [comment]
Did we change the version of Ruby used?

**Comment by IndrajeetPatil** [comment]
FWIW, this worked for `{esqlabsR}`!
 https://github.com/esqLABS/esqlabsR/pull/134

**Comment by Yuri05** [comment]
this also seems to work for ospsuite-R - just that the next problem popups immediately:
```
In download.file(p, destfile, method, mode = "wb", ...) :
  cannot open URL 'https://github.com/Open-Systems-Pharmacology/rClr/releases/download/v0.9.1/rClr_0.9.1.zip': HTTP status was '503 Egress is over the account limit.'
```
This one seems to be a GitHub problem since yesterday or so: 
https://github.com/github/feedback/discussions/8535

In the next build attempt (no code changes) the build was almost finished, but then:
```
tar xzf C:projectsospsuite-r/ospsuite_11.0.14.tar.gz -C C:/temp/ubuntu18
tar (child): C\:projectsospsuite-r/ospsuite_11.0.14.tar.gz: Cannot open: No such file or directory
```
Totally weird...

**Comment by IndrajeetPatil** [comment]
Strange, we are doing the exact same thing, and there is no issue downloading `{rClr}`: https://ci.appveyor.com/project/StephanSchaller/esqlabsr/builds/41816548

image

**Comment by Yuri05** [comment]
>Strange, we are doing the exact same thing, and there is no issue downloading {rClr}

Yep, this is a GitHub problem right now. So sometimes it works, sometimes not. Nothing we can do here.

**Comment by msevestre** [comment]
The build is still failling here

**Comment by PavelBal** [comment]
Failing because it cannot find linux versions?

> NotFound https://ci.appveyor.com/nuget/ospsuite-simmodel/Packages(Id='OSPSuite.SimModelSolver_CVODES.Ubuntu18',Version='4.1.0.8') 75ms

**Comment by Yuri05** [comment]
no, failing because of this:
```
tar xzf C:projectsospsuite-r/ospsuite_11.0.15.tar.gz -C C:/temp/ubuntu18
tar (child): C\:projectsospsuite-r/ospsuite_11.0.15.tar.gz: Cannot open: No such file or directory
```
which is the result of 
https://github.com/Open-Systems-Pharmacology/OSPSuite-R/blob/f13787ef1dee5e58edd26d911ad8423e6dee8dd7/rakefile.rb#L38-L46

Which worked all the time and suddenly not anymore.
@msevestre Ideas?

**Comment by msevestre** [comment]
the path are weird
C:projectsospsuite-r/

shouldnot it be C:projects/ospsuite-r/

**Comment by msevestre** [comment]
I am going to try to run the script locally to see if somehting has changed.

File: rakefile.rb [hunk 1]
```diff
@@ -33,16 +33,14 @@
...
   
-  # Tar file produced by the script
-  tar_file = File.join(build_dir, tar_file_name)
-
   #unzip it in a temp folder
   temp_distro_dir = File.join(temp_dir, linux_distro)
   FileUtils.mkdir_p temp_distro_dir
 
-  command_line = %W[xzf #{tar_file} -C #{temp_distro_dir}]
+  command_line = %W[xzf #{tar_file_name} -C #{temp_distro_dir}]
```
Comment (Yuri05) [inline rakefile.rb:43]: @msevestre so finally it worked after I removed the concatenation with the **build_dir**
(the current folder of the build process is apparently already the build_dir).

So somehow `File.join(build_dir, tar_file_name)` destroyed the path by removing the slashes from the **build_dir**.
Cannot explain it other than a bug in rake(?)

Comment (Yuri05) [reply rakefile.rb:43]: no it did not fully work... the linux binaries are not under artifacts anymore, rrrr

Comment (msevestre) [reply rakefile.rb:43]: >a bug in rake(?)

Lol no way . fIle.join is exactly made for this and rake has been doing this forever without problems. So unless a weird version of ruby and rake is used, I can't understand why it stopped working suddently

Comment (msevestre) [reply rakefile.rb:43]: I want to try to reproduce locally. What does build_dir look like?

Comment (Yuri05) [reply rakefile.rb:43]: build_dir is something like C:/projects/ospsuite-r
and it's the root of the cloned repor; ospsuite_x.y.zip is created also directly in the build_dir

**Comment by msevestre** [comment]
I am going to merge this because the PR is not on OSPSuite repo and I can't update the code. I think it's safe to make PR directly in the repo for you @Yuri05  :)

**Pull request merged**